alpaca/entities.go

```diff
@@ -205,6 +205,15 @@ type PortfolioHistory struct {
 
 type Side string
 
+type PositionIntent string
+
+const (
+	BuyToOpen   PositionIntent = "buy_to_open"
+	BuyToClose  PositionIntent = "buy_to_close"
+	SellToOpen  PositionIntent = "sell_to_open"
+	SellToClose PositionIntent = "sell_to_close"
+)
+
 const (
```

Comment:
Please move the `Side` constants to be just below their type declaration on line 206.

Reply:
@gcatlin I have added the change, please verfiy

Reply:
Looks good. Thanks!